src/BugsnagUnity.mm

```diff
@@ -77,25 +74,30 @@ void bugsnag_setReleaseStage(const void *configuration, char *releaseStage) {
 }
 
 void bugsnag_setNotifyReleaseStages(const void *configuration, const char *releaseStages[], int releaseStagesCount){
-  NSMutableArray *ns_releaseStages = [NSMutableArray new];
-  for (size_t i = 0; i < releaseStagesCount; i++) {
-    [ns_releaseStages addObject: [NSString stringWithUTF8String: releaseStages[i]]];
+  NSMutableSet *ns_releaseStages = [NSMutableSet new];
+  for (int i = 0; i < releaseStagesCount; i++) {
+    const char *releaseStage = releaseStages[i];
+    if (releaseStage != nil) {
+      NSString *ns_releaseStage = [NSString stringWithUTF8String: releaseStage];
+      [ns_releaseStages addObject: ns_releaseStage];
+    }
   }
-  ((__bridge BugsnagConfiguration *)configuration).notifyReleaseStages = ns_releaseStages;
+  ((__bridge BugsnagConfiguration *)configuration).enabledReleaseStages = ns_releaseStages;
 }
 
 void bugsnag_getNotifyReleaseStages(const void *configuration, const void *managedConfiguration, void (*callback)(const void *instance, const char *releaseStages[], int size)) {
```

Comment:
It looks like this method can be deleted as it's now unused, along with a few other similar ones. Here's the list of what I could spot:

- bugsnag_getApiKey
- bugsnag_getReleaseStage
- bugsnag_getNotifyReleaseStages
- bugsnag_getAppVersion
- bugsnag_getContext
- bugsnag_getNotifyUrl